modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/HiveHomologyRNASeqIntronsCheck.pm

```diff
@@ -125,6 +125,7 @@ sub fetch_input {
   }
 
   if($target_db) {
+    $target_db->dbc->disconnect_when_inactive(1);
```

Comment:
Could you please add a if statement checking if `$self->param('disconnect_jobs')` is true? Then set disconnect_when_inactive (disconnect_jobs is a parameter in the BaseRunnableDB)
There will be regions/jobs where it will be quite fast and we don't want to overload the server. If it has taken so long for someone to add the disconnect, not many jobs are longer than 8h

Reply:
Done.